docs/pages/Theming/CustomTheme.md

```diff
@@ -2,139 +2,174 @@
+import { ThemeReference, ThemeProvider } from '@fluentui-react-native/theme';
+import { createDefaultTheme } from '@fluentui-react-native/default-theme';
+import { Theme } from '@fluentui-react-native/framework';
 
-In order to have the theme override a particular component's styling, you'll need to specify a component's name and provide some overrides for the component's settings. The settings are an object that specifies how a component's tokens or slot's style are filled out.
+const theme = new ThemeReference(createDefaultTheme(),
+  () => { return {{ colors: { buttonBackground: 'red' }}}}, // overrides the buttonBackground color token, all other colors are kept in tact
+  (theme: Theme) => {
+    return {
+      { colors: { neutralBackground1: theme.colors.buttonBackground }}, // This is now red, because theme has previous recipe applied
+      { spacing: s1: '10px' }
+    }},
+  // other recipes
+);
 
-```ts
-components: {
-  <ComponentName>: {
-    <tokenName>: <overrideValue>
-  },
-},
+// Use created theme reference in ThemeProvider
+<ThemeProvider theme={theme}>
+  <App />
+</ThemeProvider>
 ```
 
-For example, if you want to change the settings on the [Button component](https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/Button/src/Button.settings.ts), you could do something like the following:
+A good example of a custom theme from our repo is [the Caterpillar theme in the Tester App](../../../apps/fluent-tester/src/FluentTester/theme/applyTheme.ts).
 
-```ts
-components: {
-  Button: {
-    tokens: {
-      backgroundColor: 'primaryButtonBackground', // Different semantic color
-      color: 'neutralBackground1', // Alias token
-    },
-    _overrides:{
-      tokens:{
-        hovered: {
-          backgroundColor: theme.host.colors['AppShade10'], // Get brand color from host
-          color: theme.host.colors['Gray96'], // Get gray from host
-          borderColor: theme.host.palette.Bkg, // Get entry from palette from host
-        },
-      },
-    }
-  },
-},
-```
+### `ThemeReference` from scratch
 
-You can see what settings can be customized by looking at a component's settings, styling, or tokens file.
+You can create your own `ThemeReference` and pass it into the `ThemeProvider`. To create a `ThemeReference` you'll need to create a new instance of `ThemeReference`. You can find the type definition [here](https://github.com/microsoft/fluentui-react-native/blob/master/packages/framework/theme/src/themeReference.ts).
 
-The advantage of this approach is that the customizations are targetted to a particular component, so the changes are localized.
+## Theme property customization
+
+There's two ways to customize a FURN theme's properties. The main way is to customize theme tokens directly by overriding certain tokens. You can also specify how to customize FURN components for the whole theme.
```

Comment:
Super Nit Opinion: When you have a certain # of choices, I always prefer a numbered list. It helps break up the text into sections and catches the eye easier, and makes a bigger impact to the reader. 

This is a personal preference, but I thought I'd mention it. Feel free to disregard if you don't agree. For example,

_There are two ways to customize a FURN theme's properties:
1. Customize theme tokens directly by overriding certain tokens (main way).
2. Specify how to customize FURN components for the whole theme._

Reply:
Thanks, that makes sense. Implemented this.